packages/api-explorer/src/components/SchemaTabs/index.jsx

```diff
@@ -93,22 +92,14 @@ export default class SchemaTabs extends Component {
   }
 
   renderSchemaExample() {
-    try {
-      const {schema} = this.state
-      let example = get(schema, EXAMPLE)
-      if (!example) {
-        example = jsf.generate(schema)
-      }
-      return (
-        <JsonViewer 
-          missingMessage={'schemaTabs.missing.example'} 
-          schema={example} 
-          key={'json-viewer-example'}
-        />
-      )
-    } catch (error) {
```

Comment:
This catch was used for throws in jsf, right?

Reply:
yes. it was used to handle the error in case of incorrect jsf generation of the scheme